environment-dev.yml

```diff
@@ -14,6 +14,7 @@ dependencies:
   - numpy
   - pandas >=2.2.0
   - platformdirs
+  - pint >=0.10,<0.24
```

Comment:
I pinned `pint` like in `xclim`, now ReadTheDocs works. Is that OK, or should I revert the changes & do a specific PR for this? I guess the latter is better, but I wanted to test this quickly to see if this would solves my test issues. @Zeitsperre  thoughts?

Reply:
This is fine for now. We'll push updates everywhere once it's ready.

Reply:
You may want to pin `numpy` while you're at it though

Reply:
Contrary to xclim, there was no lower bound, so I kept this like this, only upper bound <2.0.0